app/views/categories/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= render "shared/nav_bar"%>
+<%= render "shared/messages"%>
+<div class="d-flex justify-content-between m-1 p-1">
+  <p class= "fs-5"><%=@category.name%> category expenses:</p>
+  <p class= "fs-5"><%= number_to_currency(@transacctions.sum(:amount))%></p>
+</div>
+
+<% @transacctions.each do |item|%>
```

Comment:
- [ ] The transactions should be ordered in the most recent order. When I add a transaction it should come first on the list, not last. Kindly fix that.